templates/example_hello_world_ts/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "@apify"
```

Comment:
this should use the TS eslint config, this is the JS one, isn't it?

Reply:
Oh yeah, my bad missed that one. Fixing it now.